src/EscolaLmsCategoriesServiceProvider.php

```diff
@@ -27,5 +28,7 @@ public function boot()
     {
         $this->loadRoutesFrom(__DIR__ . '/routes.php');
         $this->loadMigrationsFrom(__DIR__ . '/../database/migrations');
+        $router = $this->app['router'];
+        $router->aliasMiddleware('role', \Spatie\Permission\Middlewares\RoleMiddleware::class);
```

Comment:
`$this->app['router']->aliasMiddleware('role', \Spatie\Permission\Middlewares\RoleMiddleware::class);`

Reply:
Zmienione